experimental/generated/beats/fields.ecs.yml

```diff
@@ -427,6 +429,7 @@
       description: Array of user roles at the time of the event.
       example: '["kibana_admin", "reporting_user"]'
       default_field: false
+    default_field: true
```

Comment:
There's quite a few of places where `default_field: true` gets inserted directly below a `default_field: false`. I assume this isn't what you're intending?

Reply:
Oooh I see, that is clearer. Thanks for that explanation!